components/renku_data_services/session/k8s_client.py

```diff
@@ -356,13 +366,13 @@ async def update_image_build_status(self, buildrun_name: str) -> models.Shipwrig
                 update=models.ShipwrightBuildStatusUpdateContent(
                     status=models.BuildStatus.failed,
                     completed_at=completion_time,
+                    error_reason=condition.reason if condition is not None else None,
                 )
             )
 
     async def cancel_image_build(self, buildrun_name: str) -> None:
```

Comment:
question: what's the reason for this method? it's a bit odd to have a method that only exists to call another method. I don't see a reason not to call cancel_build_run directly in call-sites?

Reply:
OK, will remove.

Reply:
Done in [95c6161](https://github.com/SwissDataScienceCenter/renku-data-services/pull/675/commits/95c616144bc8862699efb3a8afd6a81f2fd3f3ae).